# -- BEGIN LICENSE BLOCK ----------------------------------------------
# -- END LICENSE BLOCK ------------------------------------------------

Header header
geometry_msgs/Pose2D world_pose
bool obstructed
bool external_observations